jina/executors/__init__.py

```diff
@@ -356,7 +356,7 @@ def get_shard_workspace(
         workspace_folder: str,
         workspace_name: str,
         pea_id: int,
-        replica_id: Optional[int] = None,
+        replica_id: Optional[int] = -1,
```

Comment:
then it is not Optional.

However,  I prefer to have Optional than have a default -1 as invalid, since it is more explicit

Reply:
But I see why is needed. While I am not a fan lets keep it but then remove the Optional

Reply:
thanks, done